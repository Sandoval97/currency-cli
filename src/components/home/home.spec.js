import React from 'react';
import renderer from 'react-test-renderer';

import theme from '../../theme/theme';
import { ThemeProvider } from '@material-ui/styles';
import Home  from './index';

describe('Pruebas del componente Home', () => {
  test('Integridad del componente', () => {
    const component = renderer.create(<ThemeProvider theme={theme}><Home /></ThemeProvider>);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});