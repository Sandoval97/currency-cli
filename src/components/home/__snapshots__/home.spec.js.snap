// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas del componente Home Integridad del componente 1`] = `
<div
  className="root"
>
  <div>
    <div
      className="MuiCircularProgress-root MuiCircularProgress-colorSecondary MuiCircularProgress-indeterminate"
      role="progressbar"
      style={
        Object {
          "height": 100,
          "width": 100,
        }
      }
    >
      <svg
        className="MuiCircularProgress-svg"
        viewBox="22 22 44 44"
      >
        <circle
          className="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate"
          cx={44}
          cy={44}
          fill="none"
          r={20.2}
          strokeWidth={3.6}
          style={Object {}}
        />
      </svg>
    </div>
    <h6
      className="MuiTypography-root MuiTypography-h6"
    >
      Cargando..
    </h6>
  </div>
</div>
`;
